bpf/capabilitytracer/capability_tracer.c

```diff
@@ -0,0 +1,43 @@
+#include <bpfcore/utils.h>
+#include <gotracer/go_common.h>
+// #include <linux/sched.h>
+
+typedef struct capability_info {
+    int cap;
+    u64 pid;
+    __u8 comm[16];
+    // char comm[TASK_COMM_LEN];
+} capability_info_t;
+
+const capability_info_t *unused_2 __attribute__((unused));
+
+// Temporary tracking of capabilities
+struct {
+    __uint(type, BPF_MAP_TYPE_RINGBUF);
+    __uint(max_entries, 1 << 12);
```

Comment:
```suggestion
    __uint(max_entries, 2048);
```

rationale: let's keep this extra ring buffer smallish - we can increase the size if it starts to overflow